components/panel/panel.module.css

```diff
@@ -2,10 +2,16 @@
   background-color: var(--mantine-color-gray-0);
   height: 100%;
   padding: 0 1rem;
-  overflow-y: scroll;
-  h2 {
+  h3 {
     margin: 0;
-    font-size: 1.5rem;
+    padding: 1rem 0 0 0;
+    font-size: 1.25rem;
+  }
+  p{ 
```

Comment:
a super nit: I think we need a space between `p` and `{`

Reply:
thank you for noticing, fixed!